src/quo2/components/wallet/account_card/view.cljs

```diff
@@ -0,0 +1,105 @@
+(ns quo2.components.wallet.account-card.view
+  (:require [react-native.core :as rn]
+            [quo2.foundations.colors :as colors]
+            [quo2.components.icon :as icon]
+            [quo2.components.wallet.account-card.style :as style]
+            [quo2.components.buttons.button.view :as button]
+            [utils.i18n :as i18n]
+            [quo2.components.markdown.text :as text]
+            [quo2.theme :as theme]))
+
+(defn loading-view
+  [{:keys [customization-color type theme metrics?]}]
+  (let [watch-only? (= :watch-only type)
+        empty?      (= :empty type)]
+    [rn/view (style/card customization-color watch-only? metrics? theme)
+     [rn/view {:style {:flex-direction :row :align-items :center}}
```

Comment:
It would be beneficial if you could move more styles from the `view` namespace to the `style` one.

Reply:
agree